src/PostageStamp.sol

```diff
@@ -159,6 +174,7 @@ contract PostageStamp is AccessControl, Pausable {
         // since the block the contract was deployed, so we must supplement this batch's
         // _initialBalancePerChunk with the currentTotalOutPayment()
         uint256 normalisedBalance = currentTotalOutPayment() + (_initialBalancePerChunk);
+        require(normalisedBalance > 0, "normalisedBalance cannot be zero");
```

Comment:
This has been moved above so require is done sooner, saves a bit of gas if fails, and is better grouped with the actual value